jekyll/_cci2/concepts.adoc

```diff
@@ -305,34 +356,46 @@ The *machine executor* spins up a complete Ubuntu virtual machine image, giving
 
 [,yaml]
 ----
- version: 2.1
- jobs:
-   build1: # job name
-     docker: # Specifies the primary container image,
-     # see circleci.com/docs/circleci-images/ for
-     # the list of pre-built CircleCI images on dockerhub.
-       - image: cimg/base:2022.04-20.04
-       - image: postgres:14.2 # Specifies the database image
+version: 2.1
+
+jobs:
+  build1: # job name
+    docker: # Specifies the primary container image,
+      - image: cimg/base:2022.04-20.04
+      - image: postgres:14.2 # Specifies the database image
         # for the secondary or service container run in a common
         # network where ports exposed on the primary container are
         # available on localhost.
-         environment: # Specifies the POSTGRES_USER authentication
-          # environment variable, see circleci.com/docs/env-vars/
-          # for instructions about using environment variables.
-           POSTGRES_USER: user
-...
-   build2:
-     machine: # Specifies a machine image that uses
-     # an Ubuntu version 20.04 image.
-     # The image uses the current tag, which always points to the most recent
-     # supported release. If stability and determinism are crucial for your CI
-     # pipeline, use a release date tag with your image, e.g. ubuntu-2004:202201-02
-       image: ubuntu-2004:current
-...
-   build3:
-     macos: # Specifies a macOS virtual machine with Xcode version 12.5.1
-       xcode: "12.5.1"
- ...
+        environment: # Specifies the POSTGRES_USER authentication
+        # environment variable, see circleci.com/docs/env-vars/
+        # for instructions about using environment variables.
+          POSTGRES_USER: user
+    steps:
+      - checkout
+#...
+  build2:
+    machine: # Specifies a machine image that uses
+      # an Ubuntu version 20.04 image with Docker 20.10.12
+      # and docker-compose 1.29.2, follow CircleCI Discuss Announcements
+      # for new image releases.
+      image: ubuntu-2004:202201-02
```

Comment:
This machine image is currently being deprecated/EOLd. 
Would it be worth specifying a newer image? 

We have until Sept 2024, but if we don't want to go back and change it.

Reply:
Oh yes thanks for pointing this out. will update!